Sources/LSPLogging/Logging.swift

```diff
@@ -240,3 +244,8 @@ public class AnyLogHandler: LogHandler {
     handler(message, level)
   }
 }
+
+@available(macOS 11.0, *)
+extension os.Logger {
+  fileprivate static let shared = os.Logger()
+}
```

Comment:
I think this needs to be guarded by `#if canImport(os)`. It’s causing a compilation error on Linux.